### ISSUE TITLE
[BITV]: Replace `label` styles of `NcTextArea` with styles from `NcTextField`

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#42243]
Related: 9.1.4.3/0.1 - Note: For text (including in focused state) under 18 points (or for bold text under 14 points), the contrast ratio between the foreground and background colors should be at least 4.5:1.

https://nextcloud-vue-components.netlify.app/#/Components/NcFields?id=nctextarea

Current state (not accessible): 

![Screenshot from 2023-12-13 14-24-36](https://github.com/nextcloud/server/assets/6078378/9576aa69-f6b8-4a31-b707-b3e0292e78c4)

Styles to be replaced:

- [ ] Normal state
- [ ] Error state
- [ ] Success state
